environments/production/inventory.ini

```diff
@@ -26,9 +26,19 @@ hot_standby_server=hqstandby1.internal-va.commcarehq.org
 postgresql_replication_slots=['standby','spare']
 datavol_device=/dev/xvdb
 
+[hqdb2]
+hqdb2.internal-va.commcarehq.org
```

Comment:
can you call this pgsynclog or whatever we use for ICDS? would be good to have consistency betwee environments